docs/src/stake-pool.md

```diff
@@ -1,11 +1,427 @@
+`Gmk71cM7j2RMorRsQrsyysM4HsByQx5PuDGtDdqGLWCS`. The stake pool has full control
+over the mint.
+
+The pool creator's fee account identifier is
+`3xvXPfQi2SaTkqPV9A7BQwh4GyTe2ZPasfoaCBCnTAJ5`. When users deposit SOL into the
+stake pool, the program will transfer 3% of their contribution into this account.
+
+### Example: Create validator stake account
+
+In order to accommodate large numbers of user deposits into the stake pool, the
+stake pool only manages one stake account per validator. To add a new validator
+to the stake pool, we first create a validator-associated stake account.
+
+Looking at [validators.app](https://www.validators.app/) or other Solana validator
+lists, we choose some validators at random and start with identity
+`8SQEcP4FaYQySktNQeyxF3w8pvArx3oMEh7fPrzkN9pu` on vote account 
+`2HUKQz7W2nXZSwrdX5RkfS2rLU4j1QZLjdGCHcoUKFh3`. Let's create a validator stake account
+delegated to that vote account.
+
+```sh
+$ spl-stake-pool create-validator-stake --pool 3CLwo9CntMi4D1enHEFBe3pRJQzGJBCAYe66xFuEbmhC --validator 2HUKQz7W2nXZSwrdX5RkfS2rLU4j1QZLjdGCHcoUKFh3
+Creating stake account FYQB64aEzSmECvnG8RVvdAXBxRnzrLvcA3R22aGH2hUN
+Signature: 4pA2WKT6d2wkXEtSpiQswv22WyoFad2KX6FdPEzwBiEquvaUBEtzenys5Jh1ABPCh7yc4w8kzqMRRCwDj6ZSUV1K
+```
+
+In order to maximize censorship resistance, we want to distribute our SOL to as
+many validators as possible, so let's add a few more.
+
+```sh
+$ spl-stake-pool create-validator-stake --pool 3CLwo9CntMi4D1enHEFBe3pRJQzGJBCAYe66xFuEbmhC --validator HJiC8iJ4Sj846SswQuauFJK93UvV6zp3c2T6jzGqzhhz
+Creating stake account E5KBATUd21Dnjnh5sGFw5ngp9kdVXCcAAYMRe2WsVXie
+Signature: 4pyRZzjsWG7jP3GRZeZCo2Eb2TPjHM4kAYRFMivimme6HAee1nhzoNJBe3VSt2sv7acp5fwT7J8omBM8o3niY8gu
+$ spl-stake-pool create-validator-stake --pool 3CLwo9CntMi4D1enHEFBe3pRJQzGJBCAYe66xFuEbmhC --validator AUCzCaGAGjL3uyjFBtJs7KuJcgQWvNZu1Z2S9G3pw77G
+Creating stake account CrStLEWfme37kDc3nubK9HsmWR5dsuVUuqEKqTR4Mc5E
+Signature: 4ZUdZzUARgUCPuY8nVsJbN6vRDbVX8sYAQGYYXj2YVvjoJ2oevq2H8uzrhYApe419uoP7QYukqNstiti5p5DDukN
+$ spl-stake-pool create-validator-stake --pool 3CLwo9CntMi4D1enHEFBe3pRJQzGJBCAYe66xFuEbmhC --validator 8r1f8mwrUiYdg2Rx9sxTh4M3UAUcCBBrmRA3nxk3Z6Lm
+Creating stake account FhFft7ArhZZkh6q4ir1JZMYFgXdH6wkT5M5nmDDb1Q13
+Signature: yQqXCbuA66wQsHtkziNg3XadfZF5aCmvjfentwbZJnSPeEjJwPka3M1QY5GmR1efprptqaePn71BTMSLscX8DLr
+```
+
+NOTE: These stake accounts have not been added to the stake pool yet. Stake pools
+only accept deposits from fully delegated (warmed-up) stake accounts, so we must
+first delegate these stakes.
+
+We can see the status of stake account using the Solana command-line utility.
+
+```sh
+$ solana stake-account FYQB64aEzSmECvnG8RVvdAXBxRnzrLvcA3R22aGH2hUN
+Balance: 1.002282880 SOL
+Rent Exempt Reserve: 0.00228288 SOL
+Stake account is undelegated
+Stake Authority: 4SnSuUtJGKvk2GYpBwmEsWG53zTurVM8yXGsoiZQyMJn
+Withdraw Authority: 4SnSuUtJGKvk2GYpBwmEsWG53zTurVM8yXGsoiZQyMJn
```

Comment:
This is the first mention of `4SnSuUtJGKvk2GYpBwmEsWG53zTurVM8yXGsoiZQyMJn` and they seem to have high-authority. Who holds this key?

Reply:
I'll clarify this -- this key is the stake pool manager, which defaults to whatever is in your Solana CLI `config.yml`.  Since I was testing with one of my keys, it was chosen.